app/views/mentors/_form.html.erb

```diff
@@ -15,47 +15,39 @@
     </div>
   <% end %>
 
-  <div class="field">
-    <%= form.label :first_name %>
-    <%= form.text_field :first_name %>
-  </div>
-
-  <div class="field">
-    <%= form.label :last_name %>
-    <%= form.text_field :last_name %>
-  </div>
-
-  <div class="field">
-    <%= form.label :email %>
-    <%= form.text_field :email %>
-  </div>
-
-  <div class="field">
-    <%= form.label :manager_name %>
-    <%= form.text_field :manager_name %>
-  </div>
-
-  <div class="field">
-    <%= form.label :manager_email %>
-    <%= form.text_field :manager_email %>
-  </div>
-
-  <div class="field">
-    <%= form.label :is_active %>
-    <%= form.check_box :is_active %>
-  </div>
-
-  <div class="field">
-    <%= form.label :is_approved %>
-    <%= form.check_box :is_approved %>
-  </div>
-
-  <div class="field">
-    <%= form.label :skills %>
-    <%= form.text_area :skills %>
-  </div>
-
-  <div class="actions">
-    <%= form.submit %>
-  </div>
-<% end %>
+  <div class="card p-5" style="width: 48rem;">
+    <div class="form-group">
+      <%= form.label :first_name, class: "input-group mb-3" %>
+      <%= form.text_field :first_name, class: "form-control" %>
+    </div>
+
+    <div class="form-group">
+      <%= form.label :last_name, class: "input-group mb-3" %>
+      <%= form.text_field :last_name, class: "form-control" %>
+    </div>
+
+    <div class="form-group">
+      <%= form.label :email, class: "input-group mb-3" %>
+      <%= form.text_field :email, class: "form-control" %>
+    </div>
+
+    <div class="form-group">
+      <%= form.label :manager_name, class: "input-group mb-3" %>
+      <%= form.text_field :manager_name, class: "form-control" %>
+    </div>
+
+    <div class="form-group">
+      <%= form.label :manager_email, class: "input-group mb-3" %>
+      <%= form.text_field :manager_email, class: "form-control" %>
+    </div>
+
+    <div class="form-group">
+      <%= form.label :skills, class: "input-group mb-3" %>
+      <%= form.text_area :skills, class: "form-control" %>
+    </div>
+
```

Comment:
we dropped the checkbox here.. was this intentional? I'm going to add it back in #42 but if we don't need it then lets drop it later